fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -370,6 +370,8 @@ public class SessionVariable implements Serializable, Writable, Cloneable {
     // that, non-order-by output columns are permuted according to the ordinal column.
     public static final String FULL_SORT_LATE_MATERIALIZATION = "full_sort_late_materialization";
 
+    public static final String HDFS_BACKEND_SELECTOR_SCAN_RANGE_SHUFFLE = "hdfs_backend_selector_scan_range_shuffle";
```

Comment:
New added variable should be added in documents.

Reply:
OK

Reply:
@imay  I have made this session variable invisible.  @DorianZheng think maybe in some case we need to shuffle scan ranges before placement. In a word, this sv should be internal anyway.